SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Ingestion/ServiceCall.cs

```diff
@@ -1,54 +1,156 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading;
-using System.Threading.Tasks;
 using Microsoft.AppCenter.Ingestion.Models;
 
-namespace Microsoft.AppCenter.Ingestion.Http
+namespace Microsoft.AppCenter.Ingestion
 {
-    public abstract class ServiceCall : IServiceCall
+    /// <inheritdoc />
+    internal class ServiceCall : IServiceCall
     {
-        private CancellationTokenSource _tokenSource = new CancellationTokenSource();
+        private readonly CancellationTokenSource _tokenSource = new CancellationTokenSource();
```

Comment:
blank line between fields

Reply:
Do we use this convention here?
https://github.com/Microsoft/AppCenter-SDK-DotNet/blob/1.5.0/SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/AppCenter.cs#L20-L43

Reply:
It look wrong there too.